docs/internal/investigations/remote-kyma-runtime-configuration.md

```diff
@@ -0,0 +1,37 @@
+# Remote Kyma Runtime configuration
+
+## Introduction
+
+There is a need for configuring Dapr sidecars on a Kyma Runtime from one central Control Plane.
+
+The proposed solution is to create one central component that would hold configurations for all the Runtimes. The Runtime will have a new component, a Control Plane Runtime Agent, that will periodically fetch the configuration and apply it on the Kyma Runtime. However, the Dapr 
+sidecar have a shortcoming. As described [here](https://github.com/dapr/dapr/issues/1172), for the configuration to be applied
+the whole pod needs to restart.
+
+## Solutions
+
+There are a few solutions for this problem:
+
+**Control Plane Runtime Agent will restart every pod with a dapr sidecar**
+This solution may be the simplest, but it could add to many responsibilities for one component, it would fetch the configuration, apply it and then restart all the pods
```

Comment:
```suggestion
This solution may be the simplest, but it could add too many responsibilities for one component. A single Control Plane Runtime Agent would fetch the configuration, apply it and then restart all Pods.

```